searx/webapp.py

```diff
@@ -354,14 +355,14 @@ def get_current_theme_name(override=None):
     return theme_name
 
 
-def get_result_template(theme_name, template_name):
+def get_result_template(theme_name: str, template_name: str):
```

Comment:
Does it make sense to add the result type `-> str:` like it has been done for `get_current_theme_name() -> str:` above?  It is more a general question that fist also to other functions we have added type hints.

Reply:
Pyright does type inference, so it is smart enough to figure out that this function returns a string.